Gemfile

```diff
@@ -151,7 +151,7 @@ group :development do
   # listen higher than 3.1.1 require Ruby version >= 2.2.3 (we're currently on
   # 2.2.2). Restrict the version of `listen` to prevent `guard-rspec`
   # introducing an incompatible dependency:
-  gem 'listen', '~> 3.0.5', '<= 3.1.1'
+  gem 'listen', '>= 3.0.5', '< 3.2'
```

Comment:
@dormi did you spot this change to `.ruby-version` ?

https://github.com/dradis/dradis-ce/pull/149/files#diff-0d75b3c58057105d6c02c1fd3827050d